src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -72,8 +72,8 @@
             </v-col>
           </div>
           <div v-else>
-            <v-icon color="$BCgovGold9" small>mdi-alert</v-icon>
-            <span class="warning-text">Add Role</span>
+            <v-icon color="red darken-3" small>mdi-alert</v-icon>
+            <span class="warning-text">Missing Role</span>
```

Comment:
FYI, Tracey selected these colours to meet [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) Level AA (ie, accessibility contrast guidelines).

Reply:
Note1: the colour variable doesn't work in Vuetify components that have a color attribute.

Note2: I matched the Vuetify/Material colour name here with the declaration in theme.scss. I'm trying to reduce the number of hex colours in our template, and Tracey was gracious enough to agree to use this Material colour.

Reply:
ah interesting. So this is an app wide change? Should there be a ticket for all of it at once?

Reply:
This is the first place that we specify a "red" element, so yes it's app-wide, but no nothing else has to be updated here.

Or if you're asking about meeting the AA guidelines, yes Tracey is on it -- she is both specifying valid colours in the InVision mockups and there's a future ticket to fix all existing code.